monai/utils/misc.py

```diff
@@ -99,3 +102,48 @@ def process_bar(index, count, bar_len=30, newline=False):
     print(f"{index}/{count} {bar:s}  ", end=end)
     if index == count:
         print("")
+
+
+def manual_seed(seed):
+    _seed = seed
```

Comment:
I think it should be:
```
global _seed
_seed = seed
```
may be:
```
global _seed
_seed = int(seed)
```
(could we have a unit test for this?)

Reply:
updated in the latest patch.
Thanks.